server/src/test/java/io/crate/integrationtests/SysSnapshotsTest.java

```diff
@@ -104,13 +105,14 @@ public void testQueryAllColumns() throws Exception {
         assertThat((List<Object>) firstRow[0]).containsExactly(getFqn("tbl"));
         assertThat((List<Object>) firstRow[1]).isEmpty();
         assertThat((Long) firstRow[2]).isLessThanOrEqualTo(finishedTime);
-        assertThat(firstRow[3]).isEqualTo("s1");
-        assertThat(firstRow[4]).isEqualTo("r1");
-        assertThat((Long) firstRow[5]).isGreaterThanOrEqualTo(createdTime);
-        assertThat(firstRow[6]).isEqualTo(SnapshotState.SUCCESS.name());
-        assertThat((List<Object>) firstRow[7]).isEmpty();
-        assertThat((List<Object>) firstRow[8]).containsExactly(getFqn("tbl"));
-        assertThat(firstRow[9]).isEqualTo(Version.CURRENT.toString());
+        // firstRow[3] is UUID, not selecting it as it's random.
+        assertThat(firstRow[4]).isEqualTo("s1");
```

Comment:
Positions of other columns is different (I think we use alphabetical order currently).
Does it make it a breaking change? 

Should I instead assign ordinals according to current order and add a new column with the max ordinal? 
Will of course change ordering to rely on ordinal.